### PR TITLE
Monkeypatch fenced frame beacon response

### PR DESCRIPTION
**Pull request by linnan-github**
***
<a href="https://pr-preview.s3.amazonaws.com/linnan-github/conversion-measurement-api/pull/859.html" title="Last updated on Jun 26, 2023, 4:16 PM UTC (9517bd9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/859/8b0014b...linnan-github:9517bd9.html" title="Last updated on Jun 26, 2023, 4:16 PM UTC (9517bd9)">Diff</a>

### CODE REVIEW

**Comment by linnan-github** [comment]
@domfarolino, could you PTAL, thanks!

File: index.bs [hunk 1]
```diff
@@ -602,13 +602,18 @@ After the step
...
      is not [=list/is empty|empty=] and |contextOrigin| [=check if an origin is suitable|is suitable=]:
          1. Set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
+         1. Set |processResponse| to these steps given a [=response=] |response|:
+             1. Run [=process an attribution eligible response=] with |contextOrigin|,
+                 |attributionReportingEligibility|, and |response|.
 
          Issue: Allow "<code>[=eligibility/navigation-source=]</code>" when automatic beacon is specified.
 
-Issue: Set [=request=]'s [=request/Attribution Reporting eligibility=] to |attributionReportingEligibility|
+Issue: Set [=request=]'s [=request/Attribution Reporting eligibility=] to |attributionReportingEligibility| and
```
Comment (domfarolino) [inline index.bs:615]: Note that our fetch integration is specified now (https://github.com/WICG/fenced-frame/pull/106; https://wicg.github.io/fenced-frame/#send-a-beacon) so I think we can resolve this issue in this PR now.

Comment (linnan-github) [reply index.bs]: Thanks, updated with the fetch integration. Also added an issue to move this section to fenced frame spec.

File: index.bs [hunk 2]
```diff
@@ -602,17 +602,37 @@ After the step
...
+: [=request/Attribution Reporting eligibility=]
+:: |attributionReportingEligibility|
+
+Replace the step
+
+> [=Fetch=] |request|.
+
+with
+
+[=Fetch=] |request| with [=fetch/processResponse=] being |processResponse| if it is not null.
```
Comment (domfarolino) [inline index.bs:630]: You'll need to set useParallelQueue to true. See the note in FF spec where we fetch.

File: index.bs [hunk 3]
```diff
@@ -602,17 +602,37 @@ After the step
...
 
 </div>
 
+Issue: Move this section to <a href="https://wicg.github.io/fenced-frame/">Fenced Frame spec</a>.
```
Comment (domfarolino) [inline index.bs:634]: I'm surprised to see this given https://github.com/WICG/attribution-reporting-api/pull/848#issuecomment-1598991632. Shouldn't this ARA-specific logic just centralized in this specification?

File: index.bs (hunk 2 above)
Comment (linnan-github) [reply index.bs]: Thanks, updated.

File: index.bs (hunk 3 above)
Comment (linnan-github) [reply index.bs:637]: This is from a suggestion from @yoavweiss for the ARA intent to ship.

**Review by domfarolino — APPROVED** [review]

**Review by apasel422 — APPROVED** [review]

**Pull request merged**